.github/workflows/vanity-imports.yaml

```diff
@@ -0,0 +1,48 @@
+name: Fybrik Golang Vanity Imports
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
```

Comment:
Is it needed?

Reply:
I am not sure... I saw it in other workflows...

Reply:
Imports won't change between master and tags